roles/rsyslog/defaults/main.yaml

```diff
@@ -948,36 +961,51 @@ rsyslog__conf_viaq_elasticsearch:
               } else {
                   set $.viaq_index_name = exec_template("prefix_index_template");
               }
-          {% if openshift_logging_use_ops | default(False) %}
-              if $.viaq_index_prefix startswith "project." then {
+          {% for res in rsyslog_elasticsearch %}
```

Comment:
Should this be rsyslog_elasticsearch or perhaps better to use rsyslog_viaq ? or rsyslog_elasticsearch_viaq?

@nhosoi @richm

Reply:
I think we should make it so that the values in `rsyslog_elasticsearch` can be used for any/all elasticsearch outputs - `rsyslog_elasticsearch` and `rsyslog_viaq` and `rsyslog_elasticsearch_viaq`.

Reply:
???  Am I supposed to replace `rsyslog_elasticsearch` with `rsyslog_elasticsearch_viaq` since the current config has viaq specific setting, e.g., template="viaq_template"?  So far, we do not have a "generic" elasticsearch output for the non-viaq case as in https://github.com/linux-system-roles/logging/issues/8.  Once it is implemented, we should use `rsyslog_elasticsearch` variable?  Is that the suggestion?

Reply:
Something like that, yes.  I think we can make a more generic rsyslog_elasticsearch

Reply:
Ok.  Thanks!